assets/src/pages/webrtcInternals/WrtcInternalsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Socket } from "phoenix";
+import React, { FC, useEffect, useState } from "react";
+
+import useDetailsToggle from "./hooks/useDetailsToggle";
+
+import InternalsSection from "./InternalsSection";
+import parseIncomingStats, { Section, isChannelInput } from "./parsIncomingStats";
+import useHideVersion from "../../features/shared/hooks/useHideVersion";
+
+export const WebrtcInternalsPage: FC = () => {
+  const [chartData, setChartData] = useState<Section>({ descriptive: [], charts: [], key: "main" });
+  const { isOpen, toggle } = useDetailsToggle();
+  useHideVersion();
+
+  useEffect(() => {
+    const socket = new Socket("/socket");
+    socket.connect();
+
+    const channel = socket.channel("stats", {});
+
+    channel
+      .join()
+      .receive("ok", () => console.log("Joined the channel"))
+      .receive("error", (res) => console.error("Couldn't join the channel, ", res));
+    channel.on("metrics", ({ stats }) => {
+      if (isChannelInput(stats)) setChartData((prevStats) => parseIncomingStats(stats, prevStats, "main"));
+    });
+
+    return () => {
+      channel.leave();
+      socket.disconnect();
+    };
+  }, [chartData]);
```

Comment:
The socket doesn't depend on chartData.
Right now the channel is recreated every second
```suggestion
  }, []);
```

Reply:
I must have left it when experimenting with data fetching. It's funny linter didn't find it.